src/Core/Commander/Providers/Fetcher.js

```diff
@@ -0,0 +1,41 @@
+import { TextureLoader } from 'three';
```

Comment:
I know that THREE is present everywhere in code, but it'll be a good idea to limit its expansion in iTowns.